test/fixtures/roles.ex

```diff
@@ -0,0 +1,30 @@
+defmodule GlassFactoryApi.Fixtures.Roles do
+  def list() do
+    ~s<
+    [
+      {
+        "id": 3110,
+        "name": "partner",
+        "description": "Partner",
+        "department_id": 10,
+        "is_used": true,
+        "rate": null,
+        "cost": null,
+        "currency": null,
```

Comment:
Do we have non-nil values to put here and get a more realistic sample?

Reply:
I'm trying to see a more realistic example, but, I don't have access to that information with my profile 🙈.

Reply:
I managed to get the data 😅 , can you look again please @britto ?